java/src/main/java/com/powsybl/python/openreac/OpenReacCFunctions.java

```diff
@@ -0,0 +1,187 @@
+    public static void openReacApplyAllModifications(IsolateThread thread, ObjectHandle resultHandle,
+            ObjectHandle networkHandle, PyPowsyblApiHeader.ExceptionHandlerPointer exceptionHandlerPtr) {
+        OpenReacResult result = ObjectHandles.getGlobal().get(resultHandle);
+        Network network = ObjectHandles.getGlobal().get(networkHandle);
+        doCatch(exceptionHandlerPtr, () -> result.applyAllModifications(network));
+    }
+
+    @CEntryPoint(name = "OpenReacGetStatus")
+    public static OpenReacStatus openReacGetStatus(IsolateThread thread, ObjectHandle resultHandle,
+            PyPowsyblApiHeader.ExceptionHandlerPointer exceptionHandlerPtr) {
+        OpenReacResult result = ObjectHandles.getGlobal().get(resultHandle);
+        return doCatch(exceptionHandlerPtr, () -> OpenReacStatus.fromJavaEnum(result.getStatus()));
+    }
+
+    @CEntryPoint(name = "OpenReacGetIndicators")
+    public static StringMap openReacGetIndicators(IsolateThread thread, ObjectHandle resultHandle,
+            PyPowsyblApiHeader.ExceptionHandlerPointer exceptionHandlerPtr) {
+        OpenReacResult result = ObjectHandles.getGlobal().get(resultHandle);
+        return doCatch(exceptionHandlerPtr, () -> CTypeUtil.fromStringMap(result.getIndicators()));
+    }
+
+    @CEntryPoint(name = "runOpenReac")
+    public static ObjectHandle runOpenReac(IsolateThread thread, boolean debug, ObjectHandle networkHandle,
+            ObjectHandle paramsHandle, PyPowsyblApiHeader.ExceptionHandlerPointer exceptionHandlerPtr) {
+        Network network = ObjectHandles.getGlobal().get(networkHandle);
+        OpenReacParameters params = ObjectHandles.getGlobal().get(paramsHandle);
+
+        logger().info("Launching an OpenReac run");
+        OpenReacResult result = OpenReacRunner.run(network, network.getVariantManager().getWorkingVariantId(), params,
+                new OpenReacConfig(debug), LocalComputationManager.getDefault());
+        logger().info("OpenReac run done.");
+        return ObjectHandles.getGlobal().create(result);
+    }
+
+    @CEnum("OpenReacObjective")
+    public enum OpenReacObjective {
+        MIN_GENERATION(OpenReacOptimisationObjective.MIN_GENERATION),
+        BETWEEN_HIGH_AND_LOW_VOLTAGE_LIMIT(OpenReacOptimisationObjective.BETWEEN_HIGH_AND_LOW_VOLTAGE_LIMIT),
+        SPECIFIC_VOLTAGE_PROFILE(OpenReacOptimisationObjective.SPECIFIC_VOLTAGE_PROFILE);
+
+        public final OpenReacOptimisationObjective obj;
+
+        private OpenReacObjective(OpenReacOptimisationObjective obj) {
+            this.obj = obj;
+        }
+
+        @CEnumValue
+        public native int getCValue();
+
+        @CEnumLookup
+        public static native OpenReacObjective fromCValue(int value);
+    }
+
+    @CEnum("OpenReacStatus")
+    public enum OpenReacStatus {
+        OK,
+        NOT_OK;
+
+        public static OpenReacStatus fromJavaEnum(com.powsybl.openreac.parameters.output.OpenReacStatus obj) {
```

Comment:
Enum object are kept simple in other cases, can you write a static conversion function separated like its done in Util.java ?